plugins/inputs/socket_listener/socket_listener.go

```diff
@@ -20,6 +21,7 @@ var once sync.Once
 
 type SocketListener struct {
 	ServiceAddress string          `toml:"service_address"`
+	TimeSource     string          `toml:"time_source"`
```

Comment:
Is it okay to use the same name as in #15917 ?

Reply:
It might cause issues if the terms are not the same and checked somewhere. Either make sure there is no collision or (easier) use a different option.